reference/generators/make.rst

```diff
@@ -0,0 +1,82 @@
+.. _make_generator:
+
+make
+====
+
+.. container:: out_reference_box
+
+    This is the reference page for ``make`` generator.
+    Go to :ref:`Integrations/make<make>` if you want to learn how to integrate your project or recipes with make.
+
+This generators creates a file named *conanbuildinfo.mak* with information of dependencies in different variables that can be used for your
+make builds.
+
+Variables
+---------
+
+Variables per package. ``<PKG-NAME>`` placeholder is filled with the name of the Conan package.
+
++--------------------------------------+----------------------------------------------------------------------+
+| NAME                                 | VALUE                                                                |
++======================================+======================================================================+
+| CONAN_ROOT_<PKG-NAME>                | Absolute path to root package folder                                 |
++--------------------------------------+----------------------------------------------------------------------+
+| CONAN_SYSROOT_<PKG-NAME>             | System root folder                                                   |
++--------------------------------------+----------------------------------------------------------------------+
+| CONAN_INC_PATHS_<PKG-NAME>           | Headers folders                                                      |
```

Comment:
I've seen references to `CONAN_INC_PATHS` and `CONAN_INCLUDE_PATHS`.  Not sure what the final choice was, but need to make sure references are consistent everywhere.